middleware/csrf/csrf.go

```diff
@@ -262,8 +283,8 @@ func isFromCookie(extractor any) bool {
 // originMatchesHost checks that the origin header matches the host header
 // returns an error if the origin header is not present or is invalid
 // returns nil if the origin header is valid
-func originMatchesHost(c fiber.Ctx, trustedOrigins []string) error {
-	origin := c.Get(fiber.HeaderOrigin)
+func originMatchesHost(c fiber.Ctx, trustedOrigins []string, trustedSubOrigins []subdomain) error {
+	origin := strings.ToLower(c.Get(fiber.HeaderOrigin))
```

Comment:
can you try https://github.com/gofiber/utils/blob/master/strings.go#L8

Reply:
standard lib is faster if it is already lower case, which should be the case in most cases